ext/stringio/stringio.c

```diff
@@ -514,10 +564,11 @@ strio_set_string(VALUE self, VALUE string)
 
 /*
  * call-seq:
- *   strio.close  -> nil
+ *   close -> nil
+ *
+ * Closes +self+ for both reading and writing.
```

Comment:
I think it would be useful to keep the part about raising `IOError` when attempting to read from or write to a closed StringIO.

Reply:
Added.